scripts/legislation/africa/malawi_english.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^ ENACTED by the Parliament of Malawi as follows/,$p' | \
+    sed -n '/^ PART I -/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/' 
+}
+
+function amend_errors_in_headers {
+  sed -E 's/PART/\n\nPART/g' | \
+    sed -E 's/^(PART .*)\[•\]/\1-/' | \
+    sed -n '/PART I /,$p' | \
+    sed -E 's/Projects for which an environmen tal impact assessment is required //' 
+}
+
+function format_article_literals {
+  sed -E 's/(Article )\[([0-9]+)\] \[•\]/\n\n\(\2\)/g' | \
```

Comment:
If you're not using `Article ` in the replacement text, then you do not need to capture it (i.e. enclose it in parentheses).

Reply:
I have removed the parentheses and amended the code to be as follows: `sed -E 's/Article \[([0-9]+)\] \[•\]/\n\n\(\1\)/g' | \`